src/sections/Hero/style.module.scss

```diff
@@ -0,0 +1,62 @@
+@use "src/styles/vars.scss" as vars;
+
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
+
+.heroContainer {
+    background-image: linear-gradient(180deg, #100b44 15%, #a78590 98%);
+    min-height: 100vh;
+    font-family: Rubik, sans-serif;
+    padding-top: 10px;
+
+    .heroTitle{
+        position: absolute;
+        top: 5%;
+        z-index: 1;
+    }
+
+    .heroPlanet{
+        max-width: 60vh;
+        max-height: 60vh;
+        position: absolute;
+        top: 60%;
+        bottom: 0;
+        right: 0;
+        z-index: 1;
+    }
+
+   .heroDescription{
```

Comment:
![Screen Shot 2022-10-02 at 4 44 13 PM](https://user-images.githubusercontent.com/33165426/193481597-67a45d17-99f6-4d33-8e54-61621c9eb652.jpg)

We want to avoid hardcoding any values like this as much as we can and design with layouts in terms of relative positioning.


![Screen Shot 2022-10-02 at 4 45 39 PM](https://user-images.githubusercontent.com/33165426/193481637-65847a3b-ace7-4746-8cba-37e7acee53ef.jpg)

![Screen Shot 2022-10-02 at 4 45 32 PM](https://user-images.githubusercontent.com/33165426/193481634-d10c64e0-7d9c-4e7a-8333-883ea3859a72.jpg)

If you give the container full width, it will always center itself horizontally and we can add padding to guarantee it doesn't touch the edges.